client/src/pages/lessons/index.tsx

```diff
@@ -188,7 +202,7 @@ const LessonItem = (props: {
 
   const confirmDelete = () => {
     toast("Deleting...");
-    deleteLesson(row.node.lessonId)
+    deleteLesson(row.node.lessonId, cookies.accessToken)
```

Comment:
mostly we use `async/await` now (except maybe when inside `useEffect`)?

Any reason `confirmDelete` wouldn't be

```typescript
async function confirmDelete<Promise<void>> {
  toast("Deleting...");
  try {
    await deleteLesson(row.node.lessonId)
    onDeleted(row.node.lessonId);
    // ...
  }
  catch(err) {
    // do we toast errors?
  }
}
```